README.md

```diff
@@ -125,6 +125,9 @@ on:
   push:
     branches:
       - master
+    paths:
+      - 'docs/**.md'
+      - 'docs/images/**'
```

Comment:
Thank you for this, I think it is a good idea to add into the docs and I had completely missed it. :+1: